js/src/Ui.js

```diff
@@ -158,7 +167,8 @@ export default class Ui extends lng.Application {
     }
 
     static getImageUrl(url, opts = {}) {
-        throw new Error("{src: Ui.getImageUrl(...)} is deprecated. Please use {texture: Ui.getImage(...)} instead.");
+        //throw new Error("{src: Ui.getImageUrl(...)} is deprecated. Please use {texture: Ui.getImage(...)} instead.");
+        return this._getCdnProtocol() + "://cdn.metrological.com/image" + this.getQueryString(url, opts);
```

Comment:
Can you please remove this change from the PR?  The change in getImageUrl was only needed for the test app, and Metro officially deprecated the old usage.